test/e2e/marketplace_test.go

```diff
@@ -74,7 +71,6 @@ func MarketplaceCluster(t *testing.T) {
 
 	// get global framework variables
```

Comment:
If we are going to have multiple tests and multiple files for the tests, wouldn't it be better to move this basic workflow test into a separate file as well?

Reply:
I am planning to create another PR that would refactor the basic workflow test into it's own file. I thought it made sense to limit the scope of this PR to introducing the csc test.